include/grpc/grpc.h

```diff
@@ -44,6 +44,9 @@
 extern "C" {
 #endif
 
+#define GRPC_TRUE 1
```

Comment:
I considered using GPR_* and putting these somewhere in core/support, but I guess GRPC_* has a more public visibility.